app.config.yaml

```diff
@@ -1,4 +1,6 @@
 application:
+  hooks:
+    post-app-deploy: ./hooks/post-app-deploy.js
```

Comment:
shouldn't a folder /hooks with a post-app-deploy.js file in it exist?

Reply:
Yes, indeed. It is the other file added by this PR: [hooks/post-app-deploy.js](https://github.com/adobe/commerce-integration-starter-kit/pull/8/files#diff-9ced7fc19f71b839f150e2dc8bc71485e9eaeccb1001e8f5fcd7e872f3b3897b)